File: C/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void printObstacle(int arg1);

int main(void)
{
    int h;
    do
    {
        h = get_int("Input height: ");
    }
    while (h < 1 || h > 8);
    //prompt height 1-8, use# to build
    //each row has 2 gaps
    printObstacle(h);

}
//white space calculation if height 1, 0, if hight 2, 1,0 height 3, 2,1,0
void printObstacle(int arg1)
{
    int arg2 = arg1;
    while (arg1 > 0)
    {
        int j = 0;
        for (int i = 0; i < (arg1 - 1); i++)
        {
            printf(" ");
            j++;
        }
        //print n# arg1-j+1
        for (int i = 0; i < (arg2 - j); i++)
        {
            printf("#");
        }
        //print gap
        printf("  ");
        //print n#
        for (int i = 0; i < (arg2 - j); i++)
        {
            printf("#");
        }
        //print \n
        printf("\n");
        arg1--;
    }
}

File: python/sentimental-mario-more/mario.py
height = input("Height: ")
while not (height.isdigit() and height == ''):
    if (height.isdigit()):
        if int(height) >= 1 and int(height) <= 8:
            break
        else:
            height = input("Height: ")
    else:
        height = input("Height: ")

height = int(height)
r = 0
for h in range(height):
    for ra in range(height - r - 1):
        print(' ', end='')
    r += 1
    for a in range(r):
        print('#', end='')
    print('  ', end='')
    for a in range(r):
        print('#', end='')
    print()